src/Service/KeeperCommon.h

```diff
@@ -37,9 +37,12 @@ struct RequestForSession
     int64_t session_id;
     Coordination::ZooKeeperRequestPtr request;
 
-    /// measured in millisecond
+    /// When request create, millisecond count since system start up, used for calculate request duration
     int64_t create_time{};
```

Comment:
`create_time` is generated by `return duration_cast<std::chrono::milliseconds>(std::chrono::steady_clock::now().time_since_epoch()).count();` which is also the millisecond count since 1970-1-1.

Reply:
std::chrono::steady_clock::now().time_since_epoch().count() values can indeed differ significantly between different nodes or machines. This is because steady_clock's epoch is not tied to a specific point in real-world time and is typically initialized at the start of the system or process.